packages/client-core/src/systems/ui/PersonMenuView.tsx

```diff
@@ -96,10 +97,27 @@ const AvatarContextMenu = () => {
 
   const userState = useUserState()
   const engineState = useEngineState()
+  const authState = useAuthState()
   const user = userState.layerUsers.find((user) => user.id.value === detailState.id.value)
 
   const { t } = useTranslation()
 
+  const blockUser = () => {
+    if (authState.user?.id?.value !== null && user) {
+      const selfId = authState.user.id?.value ?? ''
+      const blockUserId = user.id?.value ?? ''
+      UserService.blockUser(selfId, blockUserId)
+    }
+  }
+
+  const addAsFriend = () => {
+    if (authState.user?.id?.value !== null && user) {
+      const selfId = authState.user.id?.value ?? ''
+      const blockUserId = user.id?.value ?? ''
```

Comment:
please change the name to `friendUserId`

Reply:
or `relatedUserId ` as used in the Service file

Reply:
@zulqarnainhanif Sure I will do that